src/extension.ts

```diff
@@ -199,6 +202,69 @@ export async function activate(context: vscode.ExtensionContext, perfStats: { lo
     ext.managedIdentityBranchDataProvider = new ManagedIdentityBranchDataProvider();
     ext.v2.api.resources.registerAzureResourceBranchDataProvider(AzExtResourceType.ManagedIdentityUserAssignedIdentities, ext.managedIdentityBranchDataProvider);
 
+
```

Comment:
Please factor this out into a new file so that extension.ts doesn't get too busy

Reply:
done.